op-supervisor/supervisor/backend/depset/static.go

```diff
@@ -124,7 +124,7 @@ func (ds *StaticConfigDependencySet) CanInitiateAt(chainID eth.ChainID, initTime
 	if !ok {
 		return false, nil
 	}
-	return initTimestamp >= dep.HistoryMinTime, nil
+	return initTimestamp > dep.HistoryMinTime, nil
```

Comment:
This will work for now, but the HistoryMinTime timestamp doesn't always align with the activation block timestamp. So this can be wrong in some configurations.

Reply:
Just for my understanding, what's the background on this change?